Cue/app/tabs/library/DeckView.js

```diff
@@ -254,6 +254,7 @@ class DeckView extends React.Component {
           onScroll={this._onScroll}>
           <DeckViewInfoHeader
             deck={this.state.deck}
+            key={this.state.deck.share_code}
```

Comment:
Setting the key forces the header to unmount and remount every time the key changes. In this case, my issue was going from private to shared, where the header wouldn't update until you popped and re-pushed `DeckView` off and back on to the Navigator stack.